.cicd/test.sh

```diff
@@ -33,6 +33,8 @@ function check_modules() {
   check_modules "--set modules.text2vec-contextionary.enabled=false --set modules.qna-transformers.enabled=true" "value: qna-transformers"
   check_modules "--set modules.text2vec-contextionary.enabled=false --set modules.img2vec-neural.enabled=true" "value: img2vec-neural"
   check_modules "--set modules.text2vec-contextionary.enabled=false --set modules.text2vec-transformers.enabled=true" "value: text2vec-transformers"
+  check_modules "--set modules.text2vec-contextionary.enabled=false --set modules.text2vec-transformers.passageQueryServices.passage.enabled=true" "value: text2vec-transformers"
+  check_modules "--set modules.text2vec-contextionary.enabled=false --set modules.text2vec-transformers.passageQueryServices.query.enabled=true" "value: text2vec-transformers"
```

Comment:
I think that this test should be changed to look for `TRANSFORMERS_PASSAGE_INFERENCE_API` and `TRANSFORMERS_QUERY_INFERENCE_API` also

Reply:
tests added